base.json

```diff
@@ -63,11 +63,15 @@
       "groupName": "prettier packages"
     },
     {
+      "description": "Update references to Docker tags for renovate/renovate in Markdown files",
       "packageNames": ["renovate/renovate"],
       "paths": [".md"],
       "extends": ["schedule:weekly"],
       "automerge": true,
-      "stabilityDays": 0
+      "stabilityDays": 0,
+      "commitMessagePrefix": "docs:",
+      "commitMessageAction": "update",
```

Comment:
```suggestion
```

this is the default https://docs.renovatebot.com/configuration-options/#commitmessageaction

Reply:
Yeah I saw that, but I thought it was better to be explicit. 😄 

If you don't like it, it's gone! 😉